frontend/src/metabase/parameters/components/ParameterValuePicker/OwnDatePicker.tsx

```diff
@@ -0,0 +1,153 @@
+import { useClickOutside } from "@mantine/hooks";
+import { useState } from "react";
+import { t } from "ttag";
+
+import { DateAllOptionsWidget } from "metabase/components/DateAllOptionsWidget";
+import { DateMonthYearWidget } from "metabase/components/DateMonthYearWidget";
+import { DateQuarterYearWidget } from "metabase/components/DateQuarterYearWidget";
+import { DateRelativeWidget } from "metabase/components/DateRelativeWidget";
+import { formatParameterValue } from "metabase/parameters/utils/formatting";
+import { Popover } from "metabase/ui";
+import type {
+  DateParameterType,
+  Parameter,
+  ParameterType,
+} from "metabase-types/api";
+
+import {
+  TextInputIcon,
+  TextInputTrirgger,
+} from "./ParameterValuePicker.styled";
+
+export function OwnDatePicker(props: {
+  value: string;
+  onValueChange: (value: string | null) => void;
+  parameter: Parameter;
+}) {
+  const { value, parameter, onValueChange } = props;
+  const [isOpen, setIsOpen] = useState(false);
+  const formatted = formatParameterValue(value, parameter);
+  const parameterType = parameter.type as DateParameterType; // TODO fix types in Parameter
+
+  const openPopover = () => setIsOpen(true);
+  const closePopover = () => setIsOpen(false);
+
+  const [triggerRef, setTriggerRef] = useState<HTMLDivElement | null>(null);
+  const dropdownRef = useClickOutside(closePopover, null, [triggerRef]);
```

Comment:
It should work out of the box, you shouldn't need this. Try `<Popover ... onChange={setIsOpen}>`